scenarioo-client/test/protractorE2E/webPages/branchAliasesPage.ts

```diff
@@ -1,113 +1,93 @@
 'use strict';
 
-import {by, element} from "protractor";
+import {by, element, ElementFinder, $, $$} from 'protractor';
+import * as Utils from '../util/util';
 
-var BaseWebPage = require('./baseWebPage'),
-    util = require('util');
+class BranchAliasesPage {
 
-function BranchAliasesPage(overridePath) {
-    if (overridePath && overridePath.length > 0) {
-        BaseWebPage.call(this, overridePath);
-    } else {
-        BaseWebPage.call(this, '/manage?tab=branchAliases');
-    }
-
-    this.branchAliasTable = element(by.css('table.table-responsive'));
-    this.saveButton = element(by.css('input.btn[value="Save"]'));
-    this.resetButton = element(by.css('input.btn[value="Reset"]'));
-
-    this.branchesDropDown = element(by.css('div.navbar ul.nav li.dropdown ul.dropdown-menu'));
+    private branchAliasTable: ElementFinder = $('table.table-responsive');
```

Comment:
thanks to type inference I guess that the type `ElementFinder` should be somehow needless here - I think we could ommit that on all such private fields, as long as we initialize them with a concrete object of correct type already (as here and usually for all such fields).

Reply:
Agree, done.